.eslintignore

```diff
@@ -1 +1,3 @@
 coverage/
+/build/**
```

Comment:
@Gwillison415 it will be very helpful to add a comment stating why there are here.